### ISSUE TITLE
forcePullImage does not pull latest image

### ISSUE THREAD

**Issue by cncloudcoder** [mesos/chronos#496]
Hi, I downloaded 2.3.4, built the code and then ran on my host. I expected every time the task begins, mesos can pull the latest image, so I added 'forcePullImage' in the 'container' part. I could see the parameter by calling chronos API '/scheduler/jobs' as below. But unfortunately the image was not updated actually on the slave. Is there anything I did wrongly?

```
container: {
    type: "docker",
    image: "cncloudcoder/chronostask:chronos-build-trusty",
    network: "HOST",
    volumes: [
       {
           hostPath: "/var/log/app",
           containerPath: "/var/log/app",
           mode: "RW"
       }
   ],
   forcePullImage: true
}
```

**Comment by dylanmei** [comment]
I've found that running `docker ps` reveals leftover containers. See #378.

**Comment by hermansc** [comment]
Are you running mesos 0.22.1?

**Comment by lancehudson** [comment]
I am running `mesos 0.22.1` and `chronos 2.3.4`. I have used the rest api and posted a job with `"forcePullImage": true` and I have pushed using the chronos-sync script.  Both times if I get from `scheduler/jobs` the json returned does NOT include `"forcePullImage": true`. I also checked the chronos state in zookeeper, it does not have force pull. I tried deleting and recreating the job, same result.
